web/src/api/ipfix.ts

```diff
@@ -62,22 +68,30 @@ export interface Fields {
   DstK8S_HostName?: string;
   /** L4 protocol */
   Proto: number;
+  /** Network interface */
+  Interface?: string;
   /** Number of packets in this flow */
   Packets: number;
+  /** In conversation tracking, A to B packets counter per conversation */
   Packets_AB?: number;
+  /** In conversation tracking, B to A packets counter per conversation */
   Packets_BA?: number;
   /** Number of bytes in this flow */
   Bytes: number;
+  /** In conversation tracking, A to B bytes counter per conversation */
   Bytes_AB?: number;
+  /** In conversation tracking, B to A bytes counter per conversation */
   Bytes_BA?: number;
   /** Start timestamp of this flow, in milliseconds */
   TimeFlowStartMs: number;
   /** End timestamp of this flow, in milliseconds */
   TimeFlowEndMs: number;
   /** Timestamp when this flow was received and processed by the flow collector, in seconds */
   TimeReceived: number;
+  /** In conversation tracking, the conversation identifier */
   _HashId?: string;
+  /** In conversation tracking, flag identifying the first flow */
   _IsFirst?: string;
+  /** In conversation tracking, a counter of flow logs per conversation */
   numFlowLogs?: number;
```

Comment:
Feel free to remove `numFlowLogs` here to hide it from doc and in `columns.ts` @jotak 
This field is not displayed yet. We can rename / change it later.